File: 백준/Silver/1475. 방 번호/방 번호.py
nums = list(map(int, input()))
result = [0] * 10
count = 0

for num in nums:
    if num == 6 or num == 9:
        count += 1
    else:
        result[num] += 1

if count % 2 == 0:
    result[6] = count // 2
else:
    result[6] = count // 2 + 1

print(max(result))

File: 백준/Silver/1475. 방 번호/방 번호.java
import java.io.*;
import java.util.Arrays;

public class Main {
	public static void main(String[] args) throws IOException {

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
		String s = br.readLine();
		int[] arr = new int[10];

		for (int i = 0; i < s.length(); i++) {
			int index = s.charAt(i) - '0';
			arr[index] += 1;
		}
		int mid = (arr[6] + arr[9] + 1) / 2;
		arr[6] = mid;
		arr[9] = mid;

		int result = Arrays.stream(arr).max().orElse(0);
		bw.write(String.valueOf(result));
		bw.flush();
		bw.close();
		br.close();
	}
}
